libs/alcotest_ext/_test_results/expect/alcotest_ext/5deda5ac4212/stdxxx

```diff
@@ -0,0 +1,2 @@
+hello
```

Comment:
those snapshot dir with hexadecimal number are not super readable. I think pytest is doing a better job here.

Reply:
I don't read the pytest snapshots very often because I find them hard to locate. The idea here is to not have to look into these files directly. Being able to consult the output of a test is important, though. The status command shows diffs between expectation and actual output. I'm thinking of adding an option to show the full output, or even show it by default (with a configurable limit of N lines). The output is already organized into long status, then short status, then stats so that the user can see what they need without rerunning anything. So, the long status section could show more.

Reply:
Our full test names (category + name) are long and contain spaces. I don't find this convenient to type, so I resort to copy-pasting and I find that copy-pasting a short hex ID is easier if we ever have to. Using nested folder to represent nested categories seems appealing but I'm not sure I'd find it practical. We could also use symlinks between ID and names. Again, I'd rather bypass the issue by having a convenient `status` command that shows us easily the output.